darts/utils/likelihood_models.py

```diff
@@ -149,7 +151,9 @@ def _distr_from_params(self, params: Tuple) -> torch.distributions.Distribution:
         pass
 
     @abstractmethod
-    def _params_from_output(self, model_output: torch.Tensor):
+    def _params_from_output(
+        self, model_output: torch.Tensor
+    ) -> Union[Tuple[torch.Tensor, ...], torch.Tensor]:
```

Comment:
in which case do we have a tuple of tensors?
I think even RNN drops the hidden_state before the likelihood

Reply:
When a distribution have several parameters, they are returned as separate Tensors (for example `GaussianLikelihood` with `mu` and `sigma`)